payjoin-client/src/main.rs

```diff
@@ -57,7 +57,7 @@ fn main() {
         .psbt;
     let psbt = load_psbt_from_base64(psbt.as_bytes()).unwrap();
     println!("Original psbt: {:#?}", psbt);
-    let pj_params = bip78::sender::Params::with_fee_contribution(bip78::bitcoin::Amount::from_sat(10000), None);
+    let pj_params = bip78::sender::Params::non_incentivizing();
```

Comment:
There's nothing against spec here, this is client-side code where client wants the library to detect the change coming from bitcoind. It's useful in simple scenarios like this one.

Reply:
ok, Let's keep it. That's useful.